app/components/primer/alpha/segmented_control.pcss

```diff
@@ -0,0 +1,170 @@
+/* SegmentedControl */
+
+.SegmentedControl {
+  display: inline-flex;
+  list-style: none;
+  background-color: var(--color-segmented-control-bg);
+  border-radius: var(--primer-borderRadius-medium, 6px);
+  box-shadow: var(--primer-borderInset-thin, inset 0 0 0 max(1px, 0.0625rem)) var(--color-border-default);
+}
+
+.SegmentedControl-item {
+  position: relative;
+  display: inline-flex;
+  border: var(--primer-borderWidth-thin, max(1px, 0.0625rem)) solid transparent;
+  border-radius: var(--primer-borderRadius-medium, 6px);
+
+  &:hover .SegmentedControl-content {
+    background-color: var(--color-segmented-control-button-hover-bg);
+    transition-duration: var(--primer-duration-fast, 80ms);
+  }
+
+  &:active .SegmentedControl-content {
+    background-color: var(--color-segmented-control-button-active-bg);
+    transition-duration: 0;
+  }
```

Comment:
`:hover` and `:active` styles shouldn't apply to active segmented control buttons.

![Kapture 2022-08-17 at 20 36 32](https://user-images.githubusercontent.com/2313998/185267194-062d1ad2-62ea-4ef6-bee6-bbf3b7bb4786.gif)